code/game/machinery/atmoalter/canister.dm

```diff
@@ -324,6 +324,19 @@ update_flag
 			else
 				release_log += "Valve was <b>opened</b> by [usr] ([usr.ckey]), starting the transfer into the <font color='red'><b>air</b></font><br>"
 				log_open()
+				//VOREStation Evil Edit
+				if(istype(src,/obj/machinery/portable_atmospherics/canister/phoron))
+					if(usr.client)
+						var/client/C = usr.client
+						if(!isnull(C.player_age))
+							if(C.player_age < 1)

```

Comment:
I assume you mean this to determine if the `player_age` is set or not, as in the case of it not being set for a server not using the database, but unset `player_age` is not null, it's the string "(Requires Database)" as per `client defines.dm:50`. By itself that is only slightly grievous except for the fact that trying to less-than a string on the next line will **always runtime** if client age is not set.

So, effectively, these changes make it impossible to perform these actions at all on any server that either has a database that is down, or does not use a database to track player age.

I can't imagine this was tested, because to not have noticed this would require you to have set up a local DB on your testing machine and overlooked this because you were using the database.